casper/src/test/scala/coop/rchain/casper/genesis/contracts/PoSSpec.scala

```diff
@@ -0,0 +1,20 @@
+package coop.rchain.casper.genesis.contracts
+
+import coop.rchain.casper.helper.RhoSpec
+import coop.rchain.rholang.build.CompiledRholangSource
+
+import scala.concurrent.duration._
+
+class PoSSpec
+    extends RhoSpec(
+      CompiledRholangSource("PoSTest.rho"),
+      Seq(
+        StandardDeploys.nonNegativeNumber,
+        StandardDeploys.makeMint,
+        StandardDeploys.authKey,
+        StandardDeploys.either,
+        StandardDeploys.revVault,
+        StandardDeploys.PoS
+      ),
```

Comment:
You'll probably need to change this based on https://github.com/rchain/rchain/pull/2328

Reply:
Done.